modules/search/class.jetpack-search-widget.php

```diff
@@ -120,20 +180,20 @@ function is_for_current_widget( $item ) {
 	 *
 	 * @since 5.7.0
 	 *
-	 * @return bool Whether the widget should display site-wide filters or not
+	 * @return bool Whether the widget should display site-wide filters or not.
 	 */
 	function should_display_sitewide_filters() {
 		$filter_widgets = get_option( 'widget_jetpack-search-filters' );
 
 		// This shouldn't be empty, but just for sanity
-		if ( empty( $filter_widgets ) )  {
+		if ( empty( $filter_widgets ) ) {
 			return false;
 		}
 
 		// If any widget has any filters, return false
 		foreach ( $filter_widgets as $number => $widget ) {
-			$widget_id = sprintf( 'jetpack-search-filters-%d', $number );
-			if ( ! empty( $widget['filters'] ) && is_active_widget( false, $widget_id, 'jetpack-search-filters' ) ) {
+			$widget_id = Jetpack_Search_Helpers::FILTER_WIDGET_BASE . $number;
```

Comment:
It looks like this change would remove the dash between `jetpack-search-filters` and `$number`.